### ISSUE TITLE
Preprocess Image

### ISSUE THREAD

**Issue by herleeyandi** [TropComplique/mtcnn-pytorch#1]
Hello, thank you so much for your great works.
1) Why do we need use this line `img = (img - 127.5)*0.0078125` in preprocess image?
2) I have try to convert it in opencv, but seems like the number of bounding box in the final is same, but in the process is much reduced. Do you think this is because of BGR to RGB conversion?, since opencv is BGR so I convert it first to RGB.
3) Why you not using `pnet. eval()` and `rnet.eval()` ?

**Comment by TropComplique** [comment]
Hi!
1. It is needed if you want to use the pre-trained models.
It is like in [the official implementation](https://github.com/kpzhang93/MTCNN_face_detection_alignment/blob/master/code/codes/MTCNNv2/detect_face.m#L15).
2. Maybe, it is because image resizing can slightly differ in `Pillow` and `opencv`.  If you want to use it with `opencv` see this implementation [pangyupo/mxnet_mtcnn_face_detection](https://github.com/pangyupo/mxnet_mtcnn_face_detection).
3. Because only `ONet` has a dropout layer inside.

**Comment by herleeyandi** [comment]
@TropComplique Thank you so much for your great works and explanation.
